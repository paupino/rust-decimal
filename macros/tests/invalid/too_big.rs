use rust_decimal_macros::dec;

fn main() {
    let _ = dec!(79_228_162_514_264_337_593_543_950_336);
}
