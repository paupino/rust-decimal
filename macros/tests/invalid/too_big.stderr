error: proc macro panicked
 --> tests/invalid/too_big.rs:4:13
  |
4 |     let _ = dec!(79_228_162_514_264_337_593_543_950_336);
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Invalid decimal: overflow from too many digits
